examples/tti/tti_operators.py

```diff
@@ -21,58 +21,64 @@ class ForwardOperator(Operator):
     `Bhaskara` uses a rational approximation.
     """
     def __init__(self, model, src, damp, data, time_order=2, spc_order=4, save=False,
-                 trigonometry='normal', **kwargs):
+                 trigonometry='normal', u_ini=None, **kwargs):
```

Comment:
initialize the wavefield to non-zero. Only used for `test_tti` so that I can do only a few time step from an already propagated wavefield.